GVFS/GVFS.Installer/CreateMacInstaller.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+CONFIGURATION=$1
+if [ -z $CONFIGURATION ]; then
+  CONFIGURATION=Debug
+fi
+
+PACKAGEVERSION=$2
+if [ -z $PACKAGEVERSION ]; then
+  PACKAGEVERSION="0.0.0.1"
+fi
+
+BUILDOUTPUTDIR=$3
+if [ -z $BUILDOUTPUTDIR ]; then
+  echo "Build output directory not specified"
+  exit 1
+fi
+
+STAGINGDIR=$BUILDOUTPUTDIR"Staging"
+VFSFORGITDESTINATION="usr/local/vfsforgit"
+KEXTDESTINATION="/Library/Extensions"
```

Comment:
Why is the kext going into this folder? Won't this cause the kext to auto-load on reboot? We're not ready for that yet.